### ISSUE TITLE
RPM package

### ISSUE THREAD

**Issue by alcir** [jaromil/HasciiCam#7]
Hello.
FYI
I successfully built an RPM package of hasciicam.
I will try to propose to include it in Fedora.

**Comment by pkovacs** [comment]
I have the request to add this project to Fedora, but it is coming from a third party.  My concern is that this project appears to be dead.   Is this the authoritative upstream location now?  Is anyone going to maintain this project as bugs are filed?   Can someone issue a proper release from a gpg-signed annotated git tag and with checksums?

**Comment by jaromil** [comment]
What makes you think this project is dead? :open_mouth: 

I'm the author and maintainer and this is the authoritative upstream location.

**Comment by pkovacs** [comment]
@jaromil Just kicking the horse to see if it bolts.  I am asked to review this package for import into Fedora which I am happy to do.   It would be helpful (just asking) if you could issue a release, e.g. 1.3 per the configure.ac  We cache the tarballs in the distro with checksums.   If you upload a public gpg key into your github profile, you can then create a gpg-signed annotated tag on the commit that corresponds to the release and then release a tarball on that tag, e.g. .tar.bz2.  Typically you would also display sha1 and/or md5 checksums on the release page.  There will be a nice green "Verified"
tag on the release.  It makes import into the distros easier, reduces concerns  about security, etc.  Please consider it.

**Comment by jaromil** [comment]
Answering myself actually, the README really needed an update and better formatting.

**Comment by jaromil** [comment]
@pkovacs signed releases are made already on our own infrastructure at https://files.dyne.org/hasciicam

I will upload my GPG key here too, however it is on pgp.mit.edu, fingerprint in the signature of any email I send, as well on the verified account on https://keybase.io/jaromil

Does that suffices? I prefer not to use github for signed releases.

**Comment by pkovacs** [comment]
@jaromil Whatever works for you! https://files.dyne.org/hasciicam/ is fine for the download area.  What is the official homepage URL of the project?

**Comment by pkovacs** [comment]
Also, can you remove the .png extension from the Icon entry in the desktop file?  Just make it `Icon=hasciicam` per the freedesktop standards which we mention here:

https://docs.fedoraproject.org/en-US/packaging-guidelines/#_icon_tag_in_desktop_files

**Comment by alcir** [comment]
> @pkovacs signed releases are made already on our own infrastructure at https://files.dyne.org/hasciicam

@jaromil but the code here on github is different from the one on dyne.org, is it?

**Comment by pkovacs** [comment]
It's ok if the official releases are slightly behind the github repository as long as we patch essential problems downstream.  I can show you how we do that.  For example, 12200d134b348a47e613e is a patch I would want to add downstream if the release doesn't reflect it.  Also need to patch the .png problem I mentioned above.

**Comment by jaromil** [comment]
I intend to pack a 1.3.1 after this exchange is concluded, especially to upstream 12200d134b348a47e613efb2944e825697206cd7

the official homepage is https://dyne.org/software/hasciicam

**Comment by alcir** [comment]
Thank you @jaromil :-)

**Comment by pkovacs** [comment]
@jaromil autotools and cmake are most often seen with c/c++.  The advantage there is the "free" portability framework you get.  We build for 6 architectures now, 5 little endian and 1 big endian, just so you know.

**Comment by pkovacs** [comment]
Probably should update the homepage in configure.ac also since http://ascii.dyne.org is not valid
```
AC_INIT([hasciicam],[1.3],[jaromil@dyne.org],[hasciicam],[http://ascii.dyne.org])
```

**Comment by jaromil** [comment]
Actually, gimme some time to do a proper update of the web presence, since we have a new shiny template at dyne and I'm stuck into one of these meetings where my attention is required 10% of the times...

**Comment by jaromil** [comment]
there we go, lets leave https://ascii.dyne.org much better and up to date. Will put a couple more new things that happened in the meantime like https://binart.eu/hasciicam/ however we are good to go for a new release. I'll need one more day for it.

**Comment by pkovacs** [comment]
@alcir when the new download is ready, ping me and we will pick this up on the Fedora review.  You will need to create a work-in-progress link somewhere for the rpm spec, e.g. on your github, so that the spec can be viewed after edits and finally approved.  

@jaromil Thanks for your help and cooperation.

Phil

**Comment by alcir** [comment]
Hey @jaromil, any news?

Thanks :-)

**Comment by jaromil** [comment]
yes, catching up. I have asked a bit around about the new webpage whats missing and if info is complete. Its good to go so just need to find a couple hours for the last touch.

**Comment by jaromil** [comment]
I'm sorry for my latency on this. Life happened. I'm now catching up a bit. I hope you are all well.

**Comment by jaromil** [comment]
Hi, made v2 with a new build system (cmake) let me know if you are interested in updating the RPM I'll give some assistance.
